README.md

```diff
@@ -56,6 +45,28 @@ See the [documentation](http://pm2.keymetrics.io/docs/usage/docker-pm2-nodejs/#u
 }
 ```
 
+### Create a `Dockerfile` in your Node.js app project
+Assuming that `pm2.json` and `package.json` are in the same folder of the `Dockerfile` and that the `src` folder contains your source code.
+
+```dockerfile
+FROM keymetrics/pm2-docker-alpine:latest
+
+# Bundle APP files
+COPY src src/
+COPY package.json .
+COPY pm2.json .
```

Comment:
Yes but I think that pm2.json It's more brand focused 😄 
What I mean is that if I a `noobie` fork a project and reads pm2.json and then google `pm2` it finds lot of useful things.
If it search for `ecosystem` that's the result:
![image](https://user-images.githubusercontent.com/3505087/28312313-02371b8a-6bb3-11e7-9d7a-c3bac19486ed.png)